### ISSUE TITLE
Search results summary doesn't meet WCAG AA contrast

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#86345]
rgba(204, 204, 204, 0.5) on #252526:

![image](https://user-images.githubusercontent.com/2193314/70192004-520ea200-16af-11ea-9dac-590f017f517d.png)

![image](https://user-images.githubusercontent.com/2193314/70192042-6b175300-16af-11ea-8115-33d61bb46418.png)

**Comment by Tyriar** [comment]
Remember to add the bug label for verification.
